mmcv/ops/csrc/tensorrt/trt_plugin.hpp

```diff
@@ -0,0 +1,5 @@
+#pragma once
```

Comment:
It would be better to have consistent c++ guard defines in headers', just like existing code in the repo. Please refer to here.